app/models/friendship.rb

```diff
@@ -0,0 +1,5 @@
+class Friendship < ApplicationRecord
+  self.primary_keys = :user_id, :friend_id
```

Comment:
`self.primary_keys = :user_id, :friend_id` :thinking:

Reply:
We are using the `composite_primary_keys` Gem for Rails to support Composite keys.
https://github.com/composite-primary-keys/composite_primary_keys#the-basics-

Reply:
I assume that your purpose is to make sure a record is not repeating. 
If that's the case then maybe this could work for you too https://devdocs.io/rails~5.0/activerecord/validations/classmethods#method-i-validates_uniqueness_of

_Not that there's anything wrong with the composite primary keys._